packages/manager/cypress/e2e/core/vpc/vpc-linodes-update.spec.ts

```diff
@@ -0,0 +1,322 @@
+
+        ui.button
+          .findByTitle('Assign Linode')
+          .should('be.visible')
+          .should('be.disabled');
+
+        mockGetLinodeConfigs(mockLinode.id, [mockConfig]).as(
+          'getLinodeConfigs'
+        );
+        cy.findByText('Linodes')
+          .should('be.visible')
+          .click()
+          .type(mockLinode.label);
+        cy.findByRole('presentation').should('be.visible').click();
+        cy.wait('@getLinodeConfigs');
+
+        mockCreateLinodeConfigInterfaces(mockLinode.id, mockConfig).as(
+          'createLinodeConfigInterfaces'
+        );
+        mockGetVPC(mockVPCAfterLinodeAssignment).as('getVPCLinodeAssignment');
+        mockGetSubnets(mockVPC.id, [mockSubnetAfterLinodeAssignment]).as(
+          'getSubnetsLinodeAssignment'
+        );
+        ui.button
+          .findByTitle('Assign Linode')
+          .should('be.visible')
+          .should('be.enabled')
+          .click();
+        cy.wait([
+          '@createLinodeConfigInterfaces',
+          '@getVPCLinodeAssignment',
+          '@getSubnetsLinodeAssignment',
+        ]);
+
+        ui.button
+          .findByTitle('Done')
+          .should('be.visible')
+          .should('be.enabled')
+          .click();
+      });
+
+    cy.get('[aria-label="View Details"]')
+      .closest('tbody')
+      .within(() => {
+        // after assigning Linode(s) to a VPC, VPC page increases number in 'Linodes' column
+        cy.findByText('1').should('be.visible');
+      });
+  });
+
+  /*
+   * - Confirms that can unassign a Linode from the VPC when feature is enabled.
+   */
+  it('can unassign Linode(s) from a VPC', () => {
+    const mockVPCs = vpcFactory.buildList(5);
+
+    const mockLinode = linodeFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+    });
+
+    const mockSecondLinode = linodeFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+    });
+
+    const mockSubnet = subnetFactory.build({
+      id: randomNumber(2),
+      label: randomLabel(),
+      linodes: [mockLinode.id, mockSecondLinode.id],
+    });
+
+    const mockVPC = vpcFactory.build({
+      id: randomNumber(),
+      label: randomLabel(),
+      subnets: [mockSubnet],
+    });
+
+    const mockConfig = linodeConfigFactory.build({
+      id: randomNumber(),
+    });
+
+    mockAppendFeatureFlags({
+      vpc: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+    mockGetVPCs(mockVPCs).as('getVPCs');
+    mockGetVPC(mockVPC).as('getVPC');
+    mockGetSubnets(mockVPC.id, [mockSubnet]).as('getSubnets');
+
+    cy.visitWithLogin('/linodes');
+    cy.wait(['@getFeatureFlags', '@getClientStream']);
+
+    ui.nav.findItemByTitle('VPC').should('be.visible').click();
+    cy.wait('@getVPCs');
+
+    cy.url().should('endWith', '/vpcs');
+
+    cy.visitWithLogin(`/vpcs/${mockVPC.id}`);
+    cy.wait(['@getVPC', '@getSubnets']);
```

Comment:
Do we have to navigate to the VPCs page manually? We might be able to save some time by navigating to `/vpcs/${mockVPC.id}` directly (I didn't test though!)

```suggestion
    cy.visitWithLogin(`/vpcs/${mockVPC.id}`);
    cy.wait(['@getFeatureFlags', '@getClientStream', '@getVPC', '@getSubnets']);
```